tile/codegen/test/tile.cc

```diff
@@ -346,19 +354,6 @@ TEST(Stencil, Pass) {
   EXPECT_THAT(data["C"], ContainerEq(kMatMulExpected));
 }
 
-TEST(Codegen, TilePassBroadcast) {
```

Comment:
Are we testing broadcast elsewhere? Is broadcast temporarily broken?

Reply:
This was an odd test, it didn't seem to have any assertions.

Also, I do have some broadcast tests elsewhere.